docs/office.md

```diff
@@ -20,3 +20,91 @@ Reference: https://api.onlyoffice.com/editors/open
 5. The file converter loads the file content from the stack
 
 ## Routes
+
+### GET /office/:id/open
+
+This route returns the parameters to open an office document. There are two
+cases:
+
+1. The document is shared and should be opened on another instance (for
+   collaborative edition)
+2. The document can be opened locally.
```

Comment:
Is that really true? Since the document is opened by the document server, we can open it "locally". I mean we can serve the DocumentEditor from the caller, right? Only the `url` and `callbackUrl` will change (if the document is mine: my cozy url, if the document is owned by someone else : cozy url of the owner)?

Reply:
> Is that really true?

I think so.

> I mean we can serve the DocumentEditor from the caller, right?

You mean for the first case, when a document is shared and you want to open the editor from the instance of the current user? In that case, we can't do that for a security reason: the CSP will block the JS (and I need them to do that, I won't try to work-around that security).

Reply:
Thanks for you answer. Makes sense to block that for a security POV. 👍